docs/community.rst

```diff
@@ -139,17 +139,27 @@ Get started!
 
    Now you can make your changes locally.
 
-6. When you're done making changes, check that your changes pass the tests::
+6. When you're done making changes, format your code with `black <https://github.com/ambv/black>`_ (black requires python >= 3.6)::
+
+    python -m pip install black
+    black .
```

Comment:
I would suggest to install `black` only and run it via `tox -e lint`  (tox and therapist will be installed via dev-requirements)